### DISCUSSION TITLE
How to implement the 'take_split' function of 'nom' in Winnow

### DISCUSSION THREAD

**Discussion by baoyachi** [winnow-rs/winnow#433] [Q&A] [answered]
See detail:https://docs.rs/nom/latest/nom/trait.InputTake.html#tymethod.take_split

**Comment by epage** [comment, marked as answer]
Isn't this `Stream::peek_slice`?

**Reply by epage** [reply]
Btw if you find we have the same function as nom but with a different name, doc-alias PRs are welcome. I did what I thought would be most useful and we can fill in gaps for what others find useful.
